crates/biome_js_formatter/tests/specs/prettier/typescript/generic/arrow-return-type.ts.snap

```diff
@@ -0,0 +1,218 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: typescript/generic/arrow-return-type.ts
+---
+
+# Input
+
+```ts
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<Descriptor> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<
+  Collections.Parts.PrintedCircuitBoardAssemblyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
+> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<Descriptor | undefined> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<
+  Collections.Parts.PrintedCircuitBoardAssembly["attributes"] | undefined
+> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<Descriptor & undefined> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<
+  Collections.Parts.PrintedCircuitBoardAssembly["attributes"] & undefined
+> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<Descriptor["attributes"]> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<
+  Collections.Parts.PrintedCircuitBoardAssembly["attributessssssssssssssssssssssss"]
+> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<keyof Descriptor> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<
+  keyof Collections.Parts.PrintedCircuitBoardAssemblyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy
+> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<Descriptor[]> => {};
+
+export const getVehicleDescriptor = async (
+  vehicleId: string
+): Promise<
+  Collections.Parts.PrintedCircuitBoardAssemblyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy[]
+> => {};
+
+```
+
+
+# Prettier differences
```

Comment:
These differences are **new**. This means that the recent changes are **lowering** our compatibility ratio. We can't accept it unless we decide to diverge on purpose.

Reply:
oooo, sorry, i got, i will fix it in a new way later